### ISSUE TITLE
XYFrame line charts stopped working in latest version

### ISSUE THREAD

**Issue by bengarvey** [emeeks/semiotic#142]
This works fine in 1.6.1, but doesn't draw the lines in 1.7.6

```
    <ResponsiveXYFrame
      size={[360,1200]}
      lines={slope}
      responsiveWidth={true}
      defined={d => showLine(d)}
      yExtent={[100,1]}
      lineDataAccessor={"data"}
      yAccessor={d => d.value}
      xAccessor={d => d.date}
      lineType={{type: 'line'}}
      hoverAnnotation={true}
      tooltipContent={ d => `${d.value} ${d.name}` }
      lineStyle={(d) => ({ stroke: d.color, strokeWidth: d.strokeWidth, opacity:'0.3' })} 
      margin={{"top":10,"bottom":40,"left":100,"right":100}}
      axes={[
        { className: 'slopeTick', orient: 'left', tickFormat: d => <text textAnchor="end" style={{fill: getColor(d-1, sortedMovies09, '2018')}}>{getName(d-1, sortedMovies09).substr(0,40)}</text>, ticks: 100},
        { className: 'slopeTick', orient: 'right', tickFormat: d => <text style={{fill: getColor(d-1, sortedMovies18, '2009')}}>{getName(d-1, sortedMovies18).substr(0,40)}</text>, ticks: 100},
        { className: 'slopeTick', orient: 'bottom', tickFormat: d => d } 
      ]}  
      hoverAnnotation={true}
    />
```

![screen shot 2018-02-13 at 8 11 00 am](https://user-images.githubusercontent.com/1342331/36151507-77b2600e-1095-11e8-8237-b914db0c31ad.png)

![screen shot 2018-02-13 at 8 10 45 am](https://user-images.githubusercontent.com/1342331/36151513-7d3b351e-1095-11e8-8582-e6bcfb8bf881.png)

Any idea?

**Comment by emeeks** [comment]
@bengarvey What does your defined function look like? Under the hood in 1.7 the data format changed slightly and now I realize what's being sent to `defined` is probably in the `data` property instead of the top-level object. If so I can push out a fix for that.

**Comment by emeeks** [comment]
The problem with how defined was dealing with data should be fixed in `1.7.7` please let me know if you're still seeing problems.

**Comment by bengarvey** [comment]
@emeeks Just tried it and your update fixed 2/3 of the three issues I'm seeing in the latest version. I'll make a ticket for the 3rd if I can't resolve it. Thanks!
http://plotdevice.bengarvey.com/
